system/jlib/jatomic.hpp

```diff
@@ -40,6 +42,30 @@ inline static void spinPause() { } // MORE: Is there an equivalent?
 inline static void spinPause() { }
 #endif
 
+//Use this class for stats which are gathered, but the values read from other threads do not need to be synchronized
```

Comment:
No, counts can never be lost, but the values read may be inconsistent.  E.g., thread 1 updates x than y, thread 2 may read an updated value of y, but an old value of x.